docs/en/integrations/data-ingestion/s3/gcs-merge-tree.md

```diff
@@ -40,12 +40,6 @@ This part of the configuration is shown in the highlighted section and specifies
                 <metadata_path>/var/lib/clickhouse/disks/gcs/</metadata_path>
 	    <!--highlight-end-->
             </gcs>
-	    <cache>
-                <type>cache</type>
-                <disk>gcs</disk>
-                <path>/var/lib/clickhouse/disks/gcs_cache/</path>
-                <max_size>10Gi</max_size>
-            </cache>
```

Comment:
I removed this because it does not make sense to have it here as `cache` is not added in `disk` section in `volumes`, then it is just redundant and confuses readers as they start to think that this will enable cache but it doesn't.